tests/Microsoft.Bot.Builder.AI.QnA.Tests/TestData/QnaMaker.EnablePrecise.json

```diff
@@ -0,0 +1,22 @@
+{
+  "activeLearningEnabled": true,
```

Comment:
rename file to make it consistent with other files: 
QnAMaker_ReturnAnswersWithPreciseAnswer.json